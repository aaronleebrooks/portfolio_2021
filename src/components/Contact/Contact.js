import styles from "./index.module.css";
import cx from "classnames";

const theme = {
  root: cx(
  ),
}

function Contact() {
  return (
    <div className={theme.root}>
      Contact Me
    </div>
  );
}

export default Contact;
