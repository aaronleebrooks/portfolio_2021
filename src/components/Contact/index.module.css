.headerBar {
    display: flex;
    align-items: center;
    justify-content: space-between;
    padding: 0 20px;
}

.navSection {
    display: flex;
}

.headerText {
    font-family: Inter,sans-serif;
    font-weight: 700;
    letter-spacing: -0.5px;
    color: rgb(22, 28, 45);
}

.largeHeaderText {
    font-size: 1.5rem;
    line-height: 1.42;
}

.smallHeaderText {
    font-size: 15px;
    line-height: 24px;
}

.navButton {
    padding: 20px;
    cursor: pointer;
}

.navButton:hover {
    color: rgb(73, 95, 239) !important;
}
