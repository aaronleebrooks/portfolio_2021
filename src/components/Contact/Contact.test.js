import { render } from '@testing-library/react';
import Contact from './Contact';

describe('Contact', () => {
  test('renders', () => {
    const { container } = render(<Contact />);
    expect(container).toMatchSnapshot();
  });
})
