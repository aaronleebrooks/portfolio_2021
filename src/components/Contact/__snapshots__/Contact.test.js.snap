// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact renders 1`] = `
<div>
  <div
    class=""
  >
    Contact Me
  </div>
</div>
`;
